rules.ini

```
@@ -37,6 +37,7 @@ AdventureGameStudio = (?:^|/)(?:AGSteam\.dll|acsetup\.cfg)$
 AGI = (?:^|/)AGIDATA\.OVL$
 Amazon_Lumberyard = ^bootstrap\.cfg$
 Asura = \.asr$
+BabylonJS = (?:^|/)(?:babylon\.js|\.babylon|lib/babylon)$
```

```suggestion
BabylonJS = (?:^|/)(?:babylon\.js|lib/babylon)$
```
`lib/babylon` is only for one game, so i'm not sure if it's worth it to have it